app/helpers/mountain_view/styleguide_helper.rb

```diff
@@ -6,5 +6,9 @@ def mv_components
         MountainView::Component.new File.basename(component_dir)
       end
     end
+
+    def render_example_stub(stubs)
+      stubs.to_yaml.gsub('---', '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.